Library/Homebrew/config.rb

```diff
@@ -35,10 +35,8 @@
 # Must use /tmp instead of $TMPDIR because long paths break Unix domain sockets
 HOMEBREW_TEMP = Pathname.new(ENV.fetch("HOMEBREW_TEMP", "/tmp"))
 
-unless defined? HOMEBREW_LIBRARY_PATH
-  # Root of the Homebrew code base
-  HOMEBREW_LIBRARY_PATH = Pathname.new(__FILE__).realpath.parent
-end
+# Root of the Homebrew code base
+HOMEBREW_LIBRARY_PATH = Pathname.new(__FILE__).realpath.parent
```

Comment:
May I propose to eliminate `HOMEBREW_LIBRARY_PATH` completely. It's quiet confusing to have it, especially we have another variable called `HOMEBREW_LIBRARY`.